cmd/server/main.go

```diff
@@ -33,27 +33,28 @@ import (
 	"github.com/abcxyz/lumberjack/internal/version"
 	"github.com/abcxyz/lumberjack/pkg/server"
 	"github.com/abcxyz/pkg/gcputil"
-	zlogger "github.com/abcxyz/pkg/logging"
+	"github.com/abcxyz/pkg/logging"
 	"github.com/abcxyz/pkg/serving"
 )
 
 func main() {
-	ctx, done := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	ctx, done := signal.NotifyContext(context.Background(),
+		syscall.SIGINT, syscall.SIGTERM)
 	defer done()
 
-	logger := zlogger.NewFromEnv("")
-	ctx = zlogger.WithLogger(ctx, logger)
+	logger := logging.NewFromEnv("")
```

Comment:
❓ Since we're refactoring, should this get a prefix now? Like `LUMBERJACK_`?

Reply:
Sure. It's a low risk change. We've been expecting this binary runs in a container so we didn't bother to add the prefix.

Reply:
I'll leave it for now, so I don't lose the approval.